feast_trino/trino_utils.py

```diff
@@ -10,9 +10,9 @@
 import trino
 from trino.dbapi import Cursor
 
-trino.constants.HEADER_USER = "X-Presto-User"
-trino.constants.HEADER_SCHEMA = "X-Presto-Schema"
-trino.constants.HEADER_CATALOG = "X-Presto-Catalog"
+trino.constants.HEADER_USER = "X-Trino-User"
```

Comment:
If we upgrade the version of Trino python client from [304](https://github.com/trinodb/trino-python-client/blob/252dc0db4e490f6dd04deffa3a4372b48c865fcc/trino/constants.py#L32) to [305](https://github.com/trinodb/trino-python-client/blob/c48f0821b313a9ea290904435f9120bd18362b44/trino/constants.py#L32) we do not need to override these constants

Reply:
I see. We might want to extend the test suite in github actions and run a matrix with multiple Trino clusters.